tools/apiview/parsers/cpp-api-parser/README.md

```diff
@@ -0,0 +1,168 @@
+# C++ Api Review Parser
+
+The C++ Api Review Parser parses the header files of a C++ project and generates an output file suitable to use with the [Api View](https://ApiView.dev)
+site.
+
+## Using the ParseAzureSdkCpp tool
+
+The simplest invocation of the tool is as follows:
+
+```powershell
+C:> ParseAzureSdkCpp -o <output-file> <input-directory>
+```
+
+where `<output-file>` is a JSON file which holds the API Review to be uploaded to the ApiView web site, and `<input-directory>` specifies the directory which contains the header files to be parsed.
+
+For instance, if your azure-sdk-for-cpp directory is located at C:\AzureCppSdk, the following will generate an API Review for the Azure Core package:
+
+```powershell
+C:> ParseAzureSdkCpp C:\AzureCppSdk\sdk\core\azure-core\inc -o core.json
+```
+
+The `ParseAzureSdkCpp` tool has the following command line switches:
+
+- `-r`, `--review` - Specifies the name of the API review for the API Review tool.
+- `--packageVersion` - Specifies the version of the package for the API Review.
+- `-o`, `--output` - Specifies the output file for the API review.
+- `--version` - Prints the version of the ParseAzureSdkCpp tool.
+- `-h`, `--help` - Prints help text about the tool.
+- `-c`, `--console` - Prints the ApiView output to the console as well as the output JSON file.
```

Comment:
Future improvement:
Display this information from tool via '-?" option.

Reply:
I took this information from the output of the tool :).